response_operations_ui/controllers/sample_controllers.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+import logging
+
+import requests
+from structlog import wrap_logger
+
+from response_operations_ui import app
+from response_operations_ui.exceptions.exceptions import ApiError
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def upload_sample(ce_id, file, total_businesses, total_ci):
```

Comment:
We don't actually need `total_businesses` or `total_ci` to upload a sample. 
It might be better to separate out building the sample summary away from this function and building it in the view maybe?

Reply:
Leaving this as it is because the submission time is linked to the upload.